README.md

```diff
@@ -474,6 +474,26 @@ Use this command to print the version of elastic-package that you have installed
 
 
 
+## Elastic Package profiles
+
+The `profiles` subcommand allows to work with different configurations. By default,
+`elastic-package` uses the "default" profile. Other profiles can be created with the
+`elastic-package profiles create` command. Once a profile is created, it will have its
+own directory inside the elastic-package data directory. Once you have more profiles,
+you can change the default with `elastic-package profiles use`.
+
+You can find the profiles in your system with `elastic-package profiles list`.
+
+You can delete profiles with `elastic-package profiles delete`.
+
+Each profile can have a `config.yml` file that allows to persist configuration settings
```

Comment:
Ergonomically, it would be nice for this to be pre-populated into the directory when the profile is created, probably also with a comment indicating that it persists and may be edited.

Reply:
I have added a `config.yml.example` that users can copy when creating their configurations.